avr/cores/megacommand/Midi/MidiSysex.hh

```diff
@@ -79,18 +79,18 @@ public:
 
   void resetRecord(uint8_t *buf = NULL, uint16_t maxLen = 0);
 
-  bool putByte(uint16_t offset, uint8_t c) {
+  inline void putByte(uint16_t offset, uint8_t c) {
```

Comment:
"Any function defined right in the class definition is implicitly marked inline."

My assumption here. Do you agree with this statement?

Reply:
According to the spec: `A function defined entirely inside a class/struct/union definition, whether it's a member function or a non-member friend function, is implicitly an inline function.`
So, yes :) I'm reverting this bit.

Reply:
Good. I was only made aware of this one recently. Some header files I have explicitly used "inline" in the past. Will need to clean this up.